sklearn/model_selection/_search.py

```diff
@@ -778,6 +828,12 @@ class GridSearchCV(BaseSearchCV):
         If "False", it is impossible to make predictions using
         this GridSearchCV instance after fitting.
 
+        For multi-metric evaluation, this needs to be a string denoting the
```

Comment:
type spec for `refit` should be updated.

perhaps add "see `scoring` parameter"